packages/patrol_cli/lib/src/common/extensions/core.dart

```diff
@@ -0,0 +1,18 @@
+extension MapX<K, V> on Map<K, V?> {
+  Map<K, V> withNullsRemoved() {
+    return {
+      for (final entry in entries)
+        if (entry.value != null) entry.key: entry.value as V,
```

Comment:
This is valid only if `V extends Object`. Otherwise this is bugged (like the flutter std)

Reply:
We do have a lint for that but it seems to not flag this: https://dart-lang.github.io/linter/lints/unrelated_type_equality_checks.html

Technically doing `entry.value != null` should raise an error, as it is not known whether `null` is an instance of `V`